opendevin/runtime/plugins/agent_skills/agentskills.py

```diff
@@ -606,12 +606,12 @@ def parse_pptx(file_path: str) -> None:
     'parse_pdf',
     'parse_docx',
     'parse_latex',
-    'parse_pptx',
-    'parse_audio',
-    'parse_video',
-    'parse_image'
+    'parse_pptx'
 ]
 
+if OPENAI_API_KEY and OPENAI_BASE_URL:
+    __all__ += ['parse_audio', 'parse_video', 'parse_image']
```

Comment:
Aha, I believe this has led to the integration test failure, but that makes sense... let me push updates to fix it